### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hariDasu**
https://github.com/typeorm/typeorm/issues/4740

thank you to @silentroach I think others can benefit from this snippet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1243,7 +1243,33 @@ The selection result will be ordered by id in descending order.
...
+
+See: https://github.com/typeorm/typeorm/blob/2c34e53bdf5bedd7e73a28066f6ad3089e9d15bb/test/github-issues/3395/entity/Post.ts#L12
+
+Sometimes your data may need to be transformed upon insertion into or reading from your database. You can use transformers to accomplish this at the level of a single record. For example, if data from your API is in JSON format and you need to `JSON.stringify(data)` on insert, and `JSON.parse(data` on read: 
+
+```
+ @Column({
+    type: "varchar",
+    nullable: true,
+    transformer: {
```
Comment (silentroach) [inline README.md]: it is already described a little bit in `docs/entities.md` and if you want to give an example of transformer usage - it is better to do it there

File: README.md [hunk 2]
```diff
@@ -1243,7 +1243,33 @@ The selection result will be ordered by id in descending order.
 The photo's albums will be left-joined and their metadata will be inner joined.
 
 You'll use the query builder in your application a lot.
-Learn more about QueryBuilder [here](./docs/select-query-builder.md).
+Learn more about QueryBuilder [here](./docs/select-query-builder.md).\
+
+
+### Using Transformers
+
+See: https://github.com/typeorm/typeorm/blob/2c34e53bdf5bedd7e73a28066f6ad3089e9d15bb/test/github-issues/3395/entity/Post.ts#L12
```
Comment (silentroach) [inline README.md]: example is enough, no reason to link a test code here

File: README.md [hunk 3]
```diff
@@ -1243,7 +1243,33 @@ The selection result will be ordered by id in descending order.
...
+
+Sometimes your data may need to be transformed upon insertion into or reading from your database. You can use transformers to accomplish this at the level of a single record. For example, if data from your API is in JSON format and you need to `JSON.stringify(data)` on insert, and `JSON.parse(data` on read: 
+
+```
+ @Column({
+    type: "varchar",
+    nullable: true,
```
Comment (silentroach) [inline README.md]: don't forget to null-check values in transform functions if column is nullable

File: README.md [hunk 4]
```diff
@@ -1243,7 +1243,33 @@ The selection result will be ordered by id in descending order.
 The photo's albums will be left-joined and their metadata will be inner joined.
 
 You'll use the query builder in your application a lot.
-Learn more about QueryBuilder [here](./docs/select-query-builder.md).
+Learn more about QueryBuilder [here](./docs/select-query-builder.md).\
```
Comment (pleerock) [inline README.md]: typo "/" at the end of the line

File: README.md [hunk 5]
```diff
@@ -1243,7 +1243,33 @@ The selection result will be ordered by id in descending order.
...
+
+
+### Using Transformers
+
+See: https://github.com/typeorm/typeorm/blob/2c34e53bdf5bedd7e73a28066f6ad3089e9d15bb/test/github-issues/3395/entity/Post.ts#L12
+
+Sometimes your data may need to be transformed upon insertion into or reading from your database. You can use transformers to accomplish this at the level of a single record. For example, if data from your API is in JSON format and you need to `JSON.stringify(data)` on insert, and `JSON.parse(data` on read:
```
Comment (pleerock) [inline README.md]: typo - missing ")"

**Comment by pleerock** [comment]
Thanks for your efforts, but changes need to be better quality. Also, we don't need to add those changes in README itself, maybe somewhere else on the site.

**Pull request closed without merging**